### PR TITLE
Add optional scree plot to report

### PR DESCRIPTION
**Pull request by WackerO**
This PR adds a scree plot to the report; it is optional, `--report_scree` is for now false by default:
[SRP254919_scree.html.zip](https://github.com/nf-core/differentialabundance/files/13297433/SRP254919_scree.html.zip)

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/differentialabundance/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/differentialabundance _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: assets/differentialabundance_report.Rmd [hunk 1]
```diff
@@ -543,6 +546,31 @@ for (assay_type in rev(names(assay_data))){
...
+  }
+}
+```
+
+```{r, echo=FALSE, results='asis', eval=params$report_scree}
+cat(paste0("\n#### Scree plot {.tabset}"))
+cat(paste0("\nThe following scree plot visualizes what percentage of total variation in the data can be explained by each of the principal components computed.\n"))
+
+for (assay_type in names(percentVar_list)) {
+  for (iv in names(percentVar_list[[assay_type]])) {
```
Comment (pinin4fjords) [inline assets/differentialabundance_report.Rmd:559]: You don't need this loop- the scree plot is not impacted by the variable and it's not used for coloring or anything, so all you're doing is making duplicate plots. One per assay type will do.

File: nextflow.config [hunk 2]
```diff
@@ -28,6 +28,7 @@ params {
     report_title               = null
     report_author              = null
     report_description         = null
+    report_scree               = false
```
Comment (pinin4fjords) [inline nextflow.config:31]: This is harmless and useful- suggest turning it on by default

**Review by pinin4fjords — CHANGES_REQUESTED** [review]
Nice! Just needs to be simplified since we have duplicate plots right now.

File: assets/differentialabundance_report.Rmd (hunk 1 above)
Comment (WackerO) [reply assets/differentialabundance_report.Rmd:559]: Oops, you are right!

**Review by pinin4fjords — APPROVED** [review]
LGTM

**Pull request merged**